compose/backend.yml

```diff
@@ -2,6 +2,35 @@
 version: '3'
 
 services:
+
+  annotation-api:
```

Comment:
rename this to mv-annotation-api, please, to make it more specific.

Reply:
Done.